R/utils_model_info.R

```diff
@@ -419,6 +420,7 @@
     is_ftest = is_ftest,
     is_meta = is_meta,
     is_wiener = inherits(x, "brmsfit") && fitfam == "wiener",
+    is_lnr = inherits(x, "brmsfit") && fitfam == "custom" && identical(brms_custom_name, "lnr"),
```

Comment:
here we can have `is_rtchoice`, and we will check for various models (LNR, LBA, ...) that will all behave the same (by having even-odd iterations)
Wiener behaves differently (negative RTs) so we need to deal separately